apps/emqx/src/emqx_limiter/src/emqx_limiter_schema.erl

```diff
@@ -56,16 +56,16 @@
 
 -import(emqx_schema, [sc/2, map/2]).
 
-roots() -> [emqx_limiter].
```

Comment:
add a namespce/0 callback, so the types e.g. `bucket` has more readable scope in the navigation panel